src/components/EventCalender/Monthly/DayView.tsx

```
@@ -0,0 +1,208 @@
+              >
+                {expanded === -100 ? 'View less' : 'View all'}
+              </button>
+            )}
+          </div>
+        </div>
+      </div>
+
+      <div className={styles.calendar_infocards}>
+        <div
+          className={styles.holidays_card}
+          role="region"
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<section>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)